test/stopAfterParseTests.sh

```diff
@@ -70,5 +70,5 @@ while read -r file; do
 		echo "$file"
 		exit 1
 	fi
-done < <(find "${REPO_ROOT}/test" -iname "*.sol" -and -not -name "documentation.sol")
+done < <(find "${REPO_ROOT}/test" -iname "*.sol" -and -not -name "documentation.sol" -and -not -name "boost_filesystem_bug.sol")
```

Comment:
This is required because the test case added in this PR does not create an actual solidity file because the test file name `==== Source://// ====` is malformed.